app/[locale]/account/profile/page.tsx

```diff
@@ -19,14 +21,16 @@ import {
 } from '@/generated/graphql';
 
 // Components
-import PageWrapper from '@/components/PageWrapper';
-import ContentContainer from '@/components/ContentContainer';
+import PageHeader from '@/components/PageHeader';
```

Comment:
Update profile page to use the new PageHeader and Layout containers and section classes